tools/autograd/gen_variable_type.py

```diff
@@ -112,6 +112,11 @@
     'quantize_per_tensor', 'quantize_per_channel'
 }
 
+# Some operators invalidate the grad_accumulator. Let's reset it.
+RESET_GRAD_ACCUMULATOR = {
+    'set'
```

Comment:
It's not called `set_` ?
Can you add `resize_` here as well please. I'm sure it suffers from the same issue :)

Reply:
`emit_body()` strips the trailing underscore for `inplace` ops:
```
base_name = name[:-1] if inplace else name[:-4] if is_out_fn else name
```

(I have verified that with `test_autograd.py` and by reading output of `print(base_name)`)

Added `resize`

Reply:
Thanks for addressing that.